src/Development/IDE/GHC/HieAst.hs

```diff
@@ -94,12 +94,12 @@ type HieM = ReaderT HieState Hsc
 mkHieFile :: ModSummary
           -> TcGblEnv
           -> RenamedSource
-          -> BS.ByteString
           -> Hsc HieFile
-mkHieFile ms ts rs src = do
+mkHieFile ms ts rs = do
   let tc_binds = tcg_binds ts
   (asts', arr) <- getCompressedAsts tc_binds rs
   let Just src_file = ml_hs_file $ ms_location ms
+  src <- liftIO $ BS.readFile src_file
```

Comment:
It is how the `mkHieFile` function is implemented in GHC 8.10. This function is called only once in the whole codebase and empty string `""` is passed as an argument to it instead of `src`. Maybe this was done for some performance/memory reasons, but I made the implementation consistent with GHC 8.10 to have fewer surprises.

Reply:
The point of copying this file is to remove this part of the implementation if I remember correctly.

Reply:
@pepeiborra Thanks for the detailed explanation. This change is reverted. The `HieAst` module for ghc-8.8 has no changes now.